src/enums/tipoBox.rb

```diff
@@ -0,0 +1,10 @@
+# Enumerado del tipo de box posibles (A, B, C o D)
```

Comment:
Creo que los boxes pueden ser de tipos diferentes, pero no de la a a la D. Según @mereloc hay dos tipos, "normal" y "refuerzo". Pero no tiene por qué ser así en todos los turnos ni en todos los servicios. En todo caso, tenéis que especificar esto en un issue, asignado a una hU.

Reply:
He creado el issue #13 para resolver este problema.